src/components/FullscreenGrid/styles.module.scss

```diff
@@ -1,15 +1,13 @@
 @import '../../scss/variables';
 
-$grid-gap: 10px;
-
 .FullscreenGrid {
   display: grid;
-  grid-gap: $grid-gap;
+  grid-gap: $default-padding;
   grid-template-areas:
     'FGHeader FGHeader FGHeader'
-    'FGNavPanel FGContentPanel FGAltPanel'
+    'FGMainSidePanel FGContent FGAltSidePanel'
```

Comment:
What's "FG" stand for?

Reply:
It stands for `FullscreenGrid`. As I understand it, `grid-template-area` names are global to all CSS. I wanted to protect this namespace a little bit but I didn't want to prefix it with `FullscreenGrid` because that's too long. The longer these names become, the harder it is to see the pattern of columns and rows.